ibr/binToJs.js

```diff
@@ -3,48 +3,42 @@
  * @author shuanglihtk@google.com (Shuang Li)
  */
 
-function swap32(val) {
-    return ((val & 0xFF) << 24)
-           | ((val & 0xFF00) << 8)
-           | ((val >> 8) & 0xFF00)
-           | ((val >> 24) & 0xFF);
-}
-
-function onFileLoad(event) {
-    var ibrData = event.target.result;
-    var scene = new THREE.Scene();
-    var camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 10000 );
-    var renderer = new THREE.WebGLRenderer();
-    renderer.setSize( window.innerWidth, window.innerHeight );
-    document.body.appendChild( renderer.domElement );
-    var lines = IBRSDK.renderLayer( ibrData );
-    for (const line of lines) {
-        scene.add(line);
-    }
-    camera.position.set( 0, 0, 7000 );
-    camera.lookAt( 0, 0, 0 );
-    animate();
+import { OrbitControls } from './node_modules/three/examples/jsm/controls/OrbitControls.js';
```

Comment:
we should try to import it from the package instead of directly accessing the file path

https://stackoverflow.com/a/52447106

Reply:
require() is not built-in for browser, only in node.js, so I don't think I can import it directly from the package in browser.